composer.json

```diff
@@ -13,20 +13,20 @@
     "require": {
         "php": "^7.1.3",
         "composer/composer": "^1.0",
-        "illuminate/console": "5.6.*|5.7.*|5.8.*|^6.0",
-        "illuminate/container": "5.6.*|5.7.*|5.8.*|^6.0",
-        "illuminate/contracts": "5.6.*|5.7.*|5.8.*|^6.0",
-        "illuminate/database": "5.6.*|5.7.*|5.8.*|^6.0",
-        "illuminate/http": "5.6.*|5.7.*|5.8.*|^6.0",
-        "illuminate/pipeline": "5.6.*|5.7.*|5.8.*|^6.0",
-        "illuminate/support": "5.6.*|5.7.*|5.8.*|^6.0",
+        "illuminate/console": "5.6.*|5.7.*|5.8.*|^6.0|^7.0",
```

Comment:
We can also drop support for 5.6 and 5.7 I think.

Reply:
😢 

I guess it only gets harder for project supporting older versions:
https://github.com/rebing/graphql-laravel/blob/223642f36e444b931ba62e57a032ffa851f9b4fe/composer.json#L37-L38

I wonder though if it's possible, similar to phpstan shim, to have something alike for larastan.